ledger/sandbox-common/BUILD.bazel

```diff
@@ -57,6 +57,47 @@ da_scala_library(
     ],
 )
 
+da_scala_library(
+    name = "sandbox-common-ce",
+    srcs = glob(["src/ce/**/*.scala"]),
+    scala_deps = [
+        "@maven//:org_scalaz_scalaz_core",
+        "@maven//:com_github_scopt_scopt",
+    ],
+    tags = ["maven_coordinates=com.daml:sandbox-common-ce:__VERSION__"],
+    visibility = ["//visibility:public"],
+    exports = [":sandbox-common"],
+    deps = [
+        ":sandbox-common",
+        "//daml-lf/data",
+        "//ledger/caching",
+        "//ledger/ledger-api-common",
+        "//ledger/participant-integration-api",
+        "//ledger/participant-state",
+        "//libs-scala/ports",
+    ],
```

Comment:
Would it be worth extracting those deps to a variable? Do we expect ce & ee to diverge here?

Reply:
Actually could this be turned into a loop like `sandbox-classic-{}` above?

Reply:
They start diverging as soon as we remove the profiler option so yes, I think we can expect this.

Reply:
Something like
```
for (edition, tags) in [("ee", []), ("ce", [...])]
```

Reply:
hm, I guess my reasoning was that there is no reason to believe that those two targets share anything apart from the name in general so I’d rather not tie them together artificially.

Reply:
👍